src/cls/2pc_queue/cls_2pc_queue_client.h

```diff
@@ -14,13 +14,33 @@
 // and more may be allocated as xattrs of the object (depending with the number of concurrent reservations)
 void cls_2pc_queue_init(librados::ObjectWriteOperation& op, const std::string& queue_name, uint64_t size);
 
-// return max capacity (bytes)
+#ifndef CLS_CLIENT_HIDE_IOCTX
```

Comment:
other cls client headers have a comment describing what this is for. if you accidentally call one of these functions within rgw, this helps you understand the compilation failure and how to fix it
```c++
// these overloads which call io_ctx.operate() or io_ctx.exec() should not be called in the rgw.
// rgw_rados_operate() should be called after the overloads w/o calls to io_ctx.operate()/exec()
#ifndef CLS_CLIENT_HIDE_IOCTX
```